addons/point_of_sale/models/account_move.py

```diff
@@ -67,6 +68,15 @@ def _compute_amount(self):
             if move.move_type == 'entry' and move.reversed_pos_order_id:
                 move.amount_total_signed = move.amount_total_signed * -1
 
+    def _inverse_payment_reference(self):
```

Comment:
Normally, for inverse we need a for loop as it is multi.  
However, here it might be simpler and clearer to just inherit the _compute_name of the account.move.line (and skip it for pos session ids receivable/payable e.g.)

Reply:
I've updated the code to remove the inverse method and added a compute method in `account.move.line`. Please review.